scene/gui/rich_text_label.cpp

```diff
@@ -4622,7 +4622,9 @@ bool RichTextLabel::search(const String &p_string, bool p_from_selection, bool p
 			queue_redraw();
 			return true;
 		}
-		p_search_previous ? current_line-- : current_line++;
+
+		if (current_line != ending_line)
+			p_search_previous ? current_line-- : current_line++;
```

Comment:
```suggestion
		if (current_line != ending_line) {
			p_search_previous ? current_line-- : current_line++;
		}
```

To follow our codestyle, conditionals need to have both curly braces around their body (with the opening one on the same line as the condition).